Doc/library/syslog.rst

```diff
@@ -60,6 +64,10 @@ The module defines the following functions:
       In previous versions, keyword arguments were not allowed, and *ident* was
       required.
 
+   .. versionchanged:: 3.12
+      Only the main interpreter is allowed to call. It will raise :exc:`RuntimeError`
+      if subinterpreters call :func:`openlog`.
```

Comment:
```suggestion
      This function is restricted in subinterpreters.
      (Only code that runs in multiple interpreters is affected and
      the restriction is not relevant for most users.)
      This may only be called in the main interpreter.
      It will raise :exc:`RuntimeError` if called in a subinterpreter.
```